src/app/code/community/Shopgate/Cloudapi/Model/Api2/Observers/WishlistsItemsRetrieve.php

```diff
@@ -41,6 +41,7 @@ public function execute(Varien_Event_Observer $observer)
         $items = array();
         /** @var Mage_Wishlist_Model_Item $item */
         foreach ($collection as $item) {
+            $this->appendChildId($item);
```

Comment:
you don't want to do magic work here by reference, it's not quite clear when reading what is mapped